anypoint-mq/3.x/modules/ROOT/pages/index.adoc

```diff
@@ -11,6 +11,9 @@ Anypoint MQ supports a wide range of messaging use cases, such as enterprise mes
 
 To use this connector, you should be familiar with Anypoint Studio, Anypoint Connectors, and application flows. The Anypoint MQ connector is free, but connecting to the service in Anypoint Platform requires a license.
 
+[IMPORTANT]
```

Comment:
Requirements and Compatibility sections look more like system/technical prerequisites. This message doesn't look right here..
Can we add it here instead since this impacts publish flow in MQ Connector? https://docs.mulesoft.com/anypoint-mq-connector/3.x/anypoint-mq-publish

Reply:
Yes, but maybe I should combine it with this #2111

Reply:
Yes that sounds good! Instead of saying latest version of MQ Connector, we should include a specific version number to avoid confusion. @somal-vora is in the process of releasing a Connector version in the next couple days. We can update the version number in the doc then.

Reply:
Sure.
Mostly likely the latest version is going to be 3.2.0. Once it's published, I will confirm.  FYI @hannanelson